tools/trace_analyzer_tool.h

```diff
@@ -164,8 +165,89 @@ struct CfUnit {
   std::map<uint32_t, uint32_t> cf_qps;
 };
 
+class TraceAnalysisResult : public TraceRecordResult {
+ public:
+  // All operations must have the same timestamp and trace type, e.g, MultiGet.
+  TraceAnalysisResult(TraceOperationType op_type, std::vector<uint32_t> cf_ids,
```

Comment:
What if in the future, we wan to get some special results from different types? E.g., for iterator, we my need to get the lower and upper bound? So, is a general TraceAnalysisResult enough?

Reply:
A general `TraceAnalysisResult` may not be enough for all potential special requirements, but at least now it's better to have individual functions with mostly the same code for each operation type.

As for iterator's lower/upper bound, they are used when creating an iterator. I guess later we may need to have some iterator creation trace type, then, the 2 bounds can be handled there.